libevm/JitVM.cpp

```diff
@@ -131,12 +132,13 @@ void getBlockHash(evm_uint256be* o_hash, evm_env* _envPtr, int64_t _number)
 	*o_hash = toEvmC(env.blockHash(_number));
 }
 
-int64_t call(
-	evm_env* _opaqueEnv,
-	evm_message const* _msg,
-	uint8_t* _outputData,
-	size_t _outputSize
-) noexcept
+void releaseVectorInPayload(evm_result const* _result)
+{
+	auto* b = reinterpret_cast<bytes const*>(&_result->payload.bytes);
+	b->~bytes();
```

Comment:
This is very unusual and most of the time dangerous. If there is no other way than to call the destructor manually, please at least add a comment.